crates/dojo/core-cairo-test/src/tests/helpers/helpers.cairo

```diff
@@ -219,13 +221,66 @@ pub mod bar {
     }
 }
 
+#[starknet::interface]
+pub trait IDojoLib<TContractState> {
+    fn sum(self: @TContractState, a: u64, b: u64) -> u128;
+}
+
+#[dojo::contract]
+pub mod dojo_lib {
+    use super::IWorldDispatcher;
+
+    #[storage]
+    struct Storage {
+        world: IWorldDispatcher,
+    }
+
+    #[abi(embed_v0)]
+    impl IDojoLibImpl of super::IDojoLib<ContractState> {
+        fn sum(self: @ContractState, a: u64, b: u64) -> u128 {
+            a.into() + b.into()
+        }
+    }
+}
+
 /// Deploys an empty world with the `dojo` namespace.
 pub fn deploy_world() -> WorldStorage {
     let namespace_def = NamespaceDef { namespace: "dojo", resources: [].span() };
 
     spawn_test_world([namespace_def].span())
 }
 
+
+/// Deploys a world with the `dojo` namespace and registers one resource
+/// of each kind.
+pub fn deploy_world_with_all_kind_of_resources() -> (WorldStorage, Span<felt252>) {
+    let namespace_def = NamespaceDef {
+        namespace: "dojo",
+        resources: [
+            TestResource::Model(m_Foo::TEST_CLASS_HASH),
+            TestResource::Event(e_SimpleEvent::TEST_CLASS_HASH),
+            TestResource::Contract(bar::TEST_CLASS_HASH),
+            TestResource::Library((dojo_lib::TEST_CLASS_HASH, @"dojolib", @"1")),
+        ]
+            .span(),
+    };
+
+    let world = spawn_test_world([namespace_def].span());
+
+    let resource_selectors = [
+        0, // world
```

Comment:
We could use the constant from `crates/dojo/core/src/world/world_contract.cairo` to make it self explanatory.